Sources/Datadog/DatadogConfiguration.swift

```diff
@@ -166,13 +154,10 @@ extension Datadog {
 
             internal var url: String {
                 switch self {
-                case .us1: return "https://trace.browser-intake-datadoghq.com/v1/input/"
+                case .us1, .us: return "https://trace.browser-intake-datadoghq.com/v1/input/"
                 case .us3: return "https://trace.browser-intake-us3-datadoghq.com/v1/input/"
-                case .eu1: return "https:/public-trace-http-intake.logs.datadoghq.eu/v1/input/"
-                case .us1_fed: return "https://trace.browser-intake-ddog-gov.com/v1/input/"
-                case .us: return "https://public-trace-http-intake.logs.datadoghq.com/v1/input/"
```

Comment:
Similar to [this change](https://github.com/DataDog/dd-sdk-ios/pull/556/files#r685072152) - new value for `.us` is changed on purpose.